### PR TITLE
breaking: use vite for dev & browser build

### PR DESCRIPTION
**Pull request by hwbllmnn**
## Description

Replaces the webpack browser build with vite. Also adds an `index.html` page and dev vite config to easily and quickly startup a dev environment.

## Do you introduce a breaking change?

- [x] Yes
- [ ] No
- [ ] I am unsure (no worries, we'll find out)

@geostyler/sprinters @geostyler/devs Please review.

### CODE REVIEW

File: src/Component/Symbolizer/ColorMapEditor/ColorMapEditor.tsx [hunk 1]
```diff
@@ -94,6 +94,7 @@ export const ColorMapEditor: React.FC<ColorMapEditorProps> = ({
         type: 'ramp'
       };
     }
+    // @ts-ignore
```
Comment (KaiVolland) [inline src/Component/Symbolizer/ColorMapEditor/ColorMapEditor.tsx:97]: Is this problem fixed with #2132 maybe?

File: vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        globals: function(name) {
+          switch (name) {
+            case 'react':
+              return 'React';
+            case 'react-dom':
+              return 'ReactDOM';
+            case 'antd':
+              return 'antd';
+            default:
+              return (name as any).replaceAll(/[/]/g, '.').replaceAll(/.*(ol\..*)/g, '$1').replaceAll(/[?].*/g, '').replaceAll(/.js$/g, '');
```
Comment (KaiVolland) [inline vite.config.ts:33]:
```suggestion
              return (name as any)
                .replaceAll(/[/]/g, '.')
                .replaceAll(/.*(ol\..*)/g, '$1')
                .replaceAll(/[?].*/g, '')
                .replaceAll(/.js$/g, '');
```

**Review by KaiVolland — APPROVED** [review]
Very nice. Did you check if the output works in different environments? Like the GeoStyler Demo, an ExtJS App, …

File: src/Component/Symbolizer/ColorMapEditor/ColorMapEditor.tsx (hunk 1 above)
Comment (hwbllmnn) [reply src/Component/Symbolizer/ColorMapEditor/ColorMapEditor.tsx:97]: Probably, I'll rebase against this once merged.

**Pull request merged**